### DISCUSSION TITLE
Running a Forest Node - Minimum requirement

### DISCUSSION THREAD

**Discussion by web3jenks** [ChainSafe/forest#3944] [Forest 🌲 Q&A] [answered]
Hey team, great to hear your update from the EngRes The Gathering. 

I am interested in running a hobby node of Forest, hardware will be ready in March. 

I can't seem to find the lowest system requirement for hardware. Do you have a recommended reference machine? 

I used to DM some team members in Filecoin slack channel before.

**Comment by LesnyRumcajs** [comment, marked as answer]
Hey @jenks-guo-filecoin! You can find Forest hardware requirements on our website: https://forest.chainsafe.io/

At the moment, it's:
```
    name: "Forest (minimal)",
    col: {
      cpu: '2-core',
      ram: '8 GiB',
      ssd: '128 GiB',
      gpu: 'none'
    }
  },
    { name: "Forest (recommended)", 
      col:{
      cpu: '4-core',
      ram: '16 GiB',
      ssd: '512 GiB (NVMe)',
      gpu: 'none'
    }},
```
If you have any trouble setting it up, I'd be glad to help!
